### PR TITLE
Change URL of the kubernetes-dashboard installation yaml file so we k…

### PR DESCRIPTION
**Pull request by laurentdavid**
I noticed a recent issue following the commit # on the kubernetes dashboard project (https://github.com/kubernetes/dashboard).
It is better to keep a track of the exact version instead of master, so I just plugged in the version information indicated in the Readme (https://github.com/kubernetes/dashboard/blob/master/README.md)

**Note:** It seems that in the master version the file is now in aio/deploy/recommended folder

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -97,7 +97,7 @@ SCRIPT
 $kubeDashScript = <<SCRIPT
 
 # Kubernetes Dashboard Setup
-kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/master/src/deploy/recommended/kubernetes-dashboard.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v1.10.1/src/deploy/recommended/kubernetes-dashboard.yaml
```
Comment (galexrt) [inline Vagrantfile]: Could you please make that into a (Makefile + Vagrantfile) variable as, e.g., seen on line 89 in this file?

File: Makefile [hunk 2]
```diff
@@ -33,6 +33,7 @@ KUBE_NETWORK ?= flannel
 
 # Addons
 K8S_DASHBOARD ?= false
+K8S_DASHBOARD_VERSION ?= v1.10.1
```
Comment (galexrt) [inline Makefile]: Could you please add it to the https://github.com/galexrt/k8s-vagrant-multi-node#variables section too.
After that LGTM and merge.

Comment (laurentdavid) [reply Makefile]: I have just done it.
Sorry, I should have noticed. Thanks !

**Comment by galexrt** [comment]
Thanks for the PR! I will also create a new tag in the repository soon.

**Pull request merged**